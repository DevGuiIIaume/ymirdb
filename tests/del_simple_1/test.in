set aa 1 2 3
set bb 4 5 6
set cc 7 8 9
list entries
del aa
list entries
get aa
del cc
list entries
del cc
set aa 1 2 3
list entries
del bb
list entries
del aa
list entries
bye
