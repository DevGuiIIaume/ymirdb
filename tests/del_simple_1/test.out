> ok

> ok

> ok

> cc [7 8 9]
bb [4 5 6]
aa [1 2 3]

> ok

> cc [7 8 9]
bb [4 5 6]

> no such key

> ok

> bb [4 5 6]

> no such key

> ok

> aa [1 2 3]
bb [4 5 6]

> ok

> aa [1 2 3]

> ok

> no entries

> bye
